packages/venia-ui/lib/components/MiniCart/miniCart.js

```diff
@@ -9,31 +9,36 @@ import defaultClasses from './miniCart.css';
 
 import { mergeClasses } from '../../classify';
 import getCurrencyCode from '../../util/getCurrencyCode';
+import { useAppContext } from '@magento/peregrine/lib/context/app';
+import { useCartContext } from '@magento/peregrine/lib/context/cart';
+import { useCheckoutContext } from '@magento/peregrine/lib/context/checkout';
 
 const MiniCart = props => {
+    const [, { closeDrawer }] = useAppContext();
+    const [
+        cartState,
+        { updateItemInCart, removeItemFromCart }
+    ] = useCartContext();
+    const [, { cancelCheckout }] = useCheckoutContext();
     const [step, setStep] = useState('cart');
+
+    const isCartEmpty =
```

Comment:
We can replace this with the derived state value when #1703 is merged.

Reply:
Not actually a merge conflict either -- I'll add a TODO here and if we remember, we remember, and if we forget, we forget :D